userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,109 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect a set of categories from a handful of images. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild.
+
+Given an image and a one-shot detector trained on category 'stop sign', the output prediction will look like:
+
+![Stop Sign vector image](images/red_stop_sign.png) ![Stop Sign prediction image](images/stop_sign_prediction_sample.png)
+
+
+#### Introductory Example
+
+In this example, the goal is to localize where in the image instances of the category 'stop sign' are present. For this task we supply a single image of a stop sign as a starter image:
+
+
+```
+import turicreate as tc
+
+# Load the starter images
+starter_images = tc.SFrame({'image':[tc.Image('stop_sign.png')],
+                   'label':['stop_sign']})
+
+# Load test images
+test_data = tc.SFrame({'image':[tc.Image('stop_sign_photo1.png'), 
+                                tc.Image('stop_sign_photo2.png')],
+                        'label':['stop_sign', 'stop_sign']})
+
+# Create a model                                      
+model = tc.one_shot_object_detector.create(starter_images, 'label')
+
+# Save predictions on the test set as a column in the SFrame
+test_data['predictions'] = model.predict(test_data)
+
+# Draw prediction bounding boxes on the test images
+test_data['image_pred'] = tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions']) 
+
+# Evaluate the model and save the results in a dictionary
+metrics = model.evaluate(test_data)
+
+# Save the model for later use in TuriCreate
+model.save('mymodel.model')
+
+# Export for use in Core ML
+model.export_coreml('MyCustomOneShotDetector.mlmodel')
+```
+
+We can preview the augmented images generated and used by the model for training:  
```

Comment:
It might be good to explain why the user might want to look at these images. I assume it's to tune hyper parameters.

Reply:
Done.